### ISSUE TITLE
composite unique indexes 

### ISSUE THREAD

**Issue by rizzi37** [isar/isar#170]
```

@Collection()
class MockModel {
  int? id;
  @Index(composite: [CompositeIndex('a2')])
  late int? a1;
  late int? a2;

  MockModel(this.a1, this.a2);


}

// expect db in 4 rows, but only 1 row.
// it there `a1 or a2` field same value, it is a unique row,
// but as i expect, `a1 with a2` is unique row.  it right ?
mockInsert() async {
  Isar isar;
  final datas = [MockModel(1,1),MockModel(1,2), MockModel(1,3), MockModel(1,4)];
  await isar.writeTxn((isar) async {
    for (var d in datas) {
      isar.mockModelSchemes.put(d);
    }
  });
}
```

**Comment by leisim** [comment]
Could you please add a little more information like a few whole sentences or some sample code? I don't quite understand what you mean with this issue.

**Comment by rizzi37** [comment]
hi @leisim  , i update with some sample code . thx~

**Comment by leisim** [comment]
Fixed in `2.1.0` 🙌
